### ISSUE TITLE
Error on using CustomerClientLinks service

### ISSUE THREAD

**Issue by SPY** [Opteo/google-ads-api#42]
Hi
I tried to use the library for creating new Manager-Client link.
I did it this way:

```javascript
const googleAdsJs = new GoogleAdsApi({
    client_id: 'xxxxxxxxxxxxxxxxxxxx.apps.googleusercontent.com',
    client_secret: 'xxxxxxxxxxxxxxxxxxxxxxxxx',
    developer_token: 'xxxxxxxxxxxxxxxxxxxx'
})

const manager = googleAdsJs.Customer({
	customer_account_id: 'xxx-xxx-xxxx',
	refresh_token: '1/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
})

manager.customerClientLinks.create({
	client_customer: 'customers/77777777777',
	status: ManagerLinkStatus.PENDING
})
```
I've got error before call: `TypeError: grpc[options.request] is not a constructor`
I found couple possible typos in `build/services/customer_client_link.js`:
```
const MUTATE_METHOD = 'mutateCustomerClientLinks';
const MUTATE_REQUEST = 'MutateCustomerClientLinksRequest';
```
and replace it with:
```
const MUTATE_METHOD = 'mutateCustomerClientLink';
const MUTATE_REQUEST = 'MutateCustomerClientLinkRequest';
```
This fix let me to go further and I got error in `service.ts`:
```
TypeError: request.setOperationsList is not a function
    at CustomerClientLinkService.<anonymous> (.../node_modules/google-ads-api/build/services/service.js:103:25)
```
Turned out CustomerClientLinkService doesn't have `setOperationsList` method, but has `setOperation`. I monkey-pathed that part as well and make call to API successfully.
```javascript
            if (!request.setOperationsList) {
                request.setOperation(operations[0]);
            }
            else {
                request.setOperationsList(operations);
            }
```

I wish I fill PR with this patch, but I'm afraid I don't understand internal machinery of the library well and I'm not sure if my fix make sense for all cases.

Best,
Ilya

**Comment by avermeil** [comment]
Hi Ilya!

Thanks for the detailed bug report, very helpful :)

The typos we already had a fix for (in https://github.com/Opteo/google-ads-api/pull/26), but we didn't know about the `request.setOperationsList()` problem.

We should have this all fixed sometime next week.

**Comment by SPY** [comment]
By the way, response is different than expected as well.
On success I got:
```
TypeError: Cannot read property 'map' of undefined
    at CustomerClientLinkService.<anonymous> (.../node_modules/google-ads-api/build/services/service.js:116:48)
```
coz response has form `{ result: { resource_name:  'customers/xxxx/customerClientLinks/xxx~yyy' } }`
but code expects `results_list` array:
```javascript
return {
                request: request.toObject(),
                partial_failure_error: response.partial_failure_error,
                results: response.results_list.map((r) => r.resourceName),
            }
```

**Comment by kritzware** [comment]
Interesting, we weren't aware of this. We'll make sure a fix for these services is released soon. 

Thanks for letting us know 👍

**Comment by avermeil** [comment]
Hey -- This issue should be fixed version 2.0.0, which we just released. Are you still experiencing these errors with this new version?

**Comment by SPY** [comment]
It works! Thank you
